src/app/events/[event]/participants/page.tsx

```diff
@@ -1,5 +1,123 @@
-export function Participants() {
+import { RoundsList } from "@/components/rounds-list";
+import { EventProps } from "../page";
+
+export default function Participants({ params }: EventProps) {
   return (
-    <h1>Participants</h1>
-  )
+    <section className="font-sans">
+      <p className="text-neutral-500 text-xs underline leading-normal">
+        Code in The Dark {params.event}
+      </p>
+      <h2 className="text-2xl text-neutral-900 leading-normal font-bold mt-0.5">
+        Gerenciar Chave
+      </h2>
+      <RoundsList rounds={rounds} />
```

Comment:
Please, use the same component design that I've suggested for the participants list: Break into more components and make it flexible.

Reply:
Done!

Reply:
Have you changed this part? I've just checked it's still the same code 🤔

Reply:
Now it's done